docs/release-taxonomy.md

```diff
@@ -0,0 +1,31 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+Releases at Hyperledger are done according to the [semver.org](https://semver.org) [taxonomy](#1). In addition to the semantic versioning scheme, where we use MAJOR.MINOR.PATCH numbering, following the established guidance given in the semver specification, it is also strongly encouraged that projects should use the following tags, as permitted by semver:
+
+* [preview](#preview)
+* [alpha](#alpha)
+* [beta](#beta)
+* [rcN](#rcn) (release candidate N - where N is 1-n incremented for each candidate)
+* [snapshot](#2)-&lt;sha&gt; for interim, possible unstable builds
+
+## preview
+
+Not feature complete, but most functionality is implemented. Any missing functionality that is committed to the production release is identified, and there are specific people working on those features. Not heavily performance tuned, but performance testing has started with the first few hotspots identified and perhaps even addressed. No highest priority issues are in an open state. First-level developer documentation provided to help new developers with the learning curve.
+
+## alpha
+
+Feature complete, for all features committed to the production release. Ready for Proof of Concept-level deployments. Performance can be characterized in a predictable way, so that basic PoC's can be done within the bounds of published expectations. APIs are documented. First attempts at end-user documentation have been made. Developer documentation is further advanced. No highest priority issues are in an open state.
+
+## beta
+
+Feature complete for all features committed to the production release, perhaps with optional features when safe to add. Ready for Pilot-level engagements. Performance is very well characterized and active optimizations are being done, with a target set for the Production release. No highest priority or high priority bugs are in an open state. Developer documentation is complete; end-user documentation is mostly done.
+
+## rcN
+
+For a forthcoming production release, we will prepare multiple candidate releases intended to be heavily tested by the community. As we cycle through resolving uncovered issues, we will issue another when no remaining highest or high priority issues remain, and repeat until we feel confident in releasing a production release, with no qualifier. e.g. 1.0.
+
+## 1
+Note that this is **not** about exiting from the incubator, though a beta or production release should not be made while still in incubation.
```

Comment:
`though a beta or production release should not be made while still in incubation`

Is this still valid with our latest decisions?

Reply:
Good question - @hyperledger/tsc ?

Reply:
Looking at the [decision log](https://wiki.hyperledger.org/display/TSC/TSC+Decision+Log) I see the following under the decision for Hyperledger Explorer major release v1.0.0
> Projects can now produce major releases at their discretion, including while in Incubation. See [2020 01 30 TSC Minutes](https://wiki.hyperledger.org/display/TSC/2020+01+30+TSC+Minutes).

Reply:
so, I strike everything including and after the /,.*$/s//. ?

Reply:
I explored a rewording

Reply:
That seems right to me.